packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -117,58 +116,58 @@ class Wizard extends React.Component<WizardProps> {
     hasBodyPadding: true
   };
 
-  public container = null;
+  public container?: HTMLDivElement = undefined;
   public titleId = `pf-wizard-title-0`;
   public descriptionId = `pf-wizard-description-0`;
 
-  constructor(props) {
+  constructor(props: WizardProps) {
     super(props);
-    const newId = currentId++;
+    const newId = Wizard.currentId++;
     this.titleId = `pf-wizard-title-${newId}`;
     this.descriptionId = `pf-wizard-description-${newId}`;
   }
 
   public state = {
-    currentStep: Number.isInteger(this.props.startAtStep) ? this.props.startAtStep : 1,
+    currentStep: this.props.startAtStep && Number.isInteger(this.props.startAtStep) ? this.props.startAtStep : 1,
     isNavOpen: false
   };
 
-  private handleKeyClicks = event => {
+  handleKeyClicks(event: KeyboardEvent): void {
     if (event.keyCode === KEY_CODES.ESCAPE_KEY) {
       if (this.state.isNavOpen) {
         this.setState({ isNavOpen: !this.state.isNavOpen })
       } else if (this.props.isOpen) {
-        this.props.onClose();
+        this.props.onClose!();
       }
     }
   };
 
-  private toggleSiblingsFromScreenReaders = hide => {
+  toggleSiblingsFromScreenReaders(hide: boolean): void {
     const bodyChildren = document.body.children;
     for (const child of Array.from(bodyChildren)) {
       if (child !== this.container) {
-        hide ? child.setAttribute('aria-hidden', hide) : child.removeAttribute('aria-hidden');
+        hide ? child.setAttribute('aria-hidden', '' + hide) : child.removeAttribute('aria-hidden');
       }
     }
   };
 
   public componentDidMount() {
-    document.body.appendChild(this.container);
+    if (this.container) {
+      document.body.appendChild(this.container);
+    }
     this.toggleSiblingsFromScreenReaders(true);
     document.addEventListener('keydown', this.handleKeyClicks, false);
   }
 
   public componentWillUnmount() {
-    document.body.removeChild(this.container);
+    if (this.container) {
+      document.body.removeChild(this.container);
+    }
     this.toggleSiblingsFromScreenReaders(false);
     document.removeEventListener('keydown', this.handleKeyClicks, false);
   }
 
-  private onNavToggle = isNavOpen => {
-    this.setState({ isNavOpen })
-  }
-
-  private onNext = () => {
+  onNext(): void {
```

Comment:
arrow functions bind `this` automatically , if this is not arrow function i think we need to bind in constructor
`this.onNext = this.onNext.bind(this);`

Reply:
You're right, fixed in latest commit